src/game/scenes/mainmenu/menu_configuration.c

```diff
@@ -41,7 +47,8 @@ component *menu_configuration_create(scene *s) {
     component *menu = menu_create(11);
     menu_attach(menu, label_create(&tconf, "CONFIGURATION"));
     menu_attach(menu, filler_create());
-    menu_attach(menu, filler_create());
+    menu_attach(menu,
+                textbutton_create(&tconf, "LANGUAGE", "Forstar du ikke engelsk?", COM_ENABLED, menu_enter_language, s));
```

Comment:
Lol, moonspeak ;)

Reply:
It's norwegian!

Reply:
Well, better than danish, I guess ... ;)

Reply:
Added Danish translation (Dansk Maskinoversottelse)